docs/learning/WriteAConnector.md

```diff
@@ -555,49 +488,26 @@ For background on when to use a dynamic schema, please checkout [Dynamic Schemas
 This method is the main workhorse of your Connector. When UpdateExistingData is called, models and definitions should be created and available for insertion of elements (if that is the desired workflow). Note: In the example below, definition elements are being inserted into the definition model at this point as well. Physical elements and Group elements can now be converted.
 
 ```ts
-  public async updateExistingData() {
-    const groupModelId = this.queryGroupModel();
-    const physicalModelId = this.queryPhysicalModel();
-    const definitionModelId = this.queryDefinitionModel();
-    if (undefined === groupModelId || undefined === physicalModelId || undefined === definitionModelId) {
-const error =`Unable to find model Id for ${undefined === groupModelId ? ModelNames.Group : (undefined === physicalModelId ? ModelNames.Physical : ModelNames.Definition)}`;
-      throw new IModelError(IModelStatus.BadArg, error, Logger.logError, loggerCategory);
-    }
-
-    if (this._sourceDataState === ItemState.Unchanged) {
-      return;
-    }
-
-    if (this._sourceDataState === ItemState.New) {
-      this.insertCategories();
-      this.insertMaterials();
-      this.insertGeometryParts();
-    }
-
-    this.convertGroupElements(groupModelId);
-    this.convertPhysicalElements(physicalModelId, definitionModelId, groupModelId);
-    this.synchronizer.imodel.views.setDefaultViewId(this.createView(definitionModelId, physicalModelId, "TestBridgeView"));
-  }
-
+[[include:TestConnector-updateExistingData.example-code]]
 ```
 
 ### Execution Sequence
 
-The ultimate purpose of a Connector is to synchronize an iModel with the data in one or more source documents. The synchronization step involves authorization, communicating with an iModel server, converting data, and concurrency control. iTwin.js defines a framework in which the Connector itself can focus on the tasks of extraction, alignment, and change-detection. The other functions are handled by classes provided by iTwin.js. The framework is implemented by the BridgeRunner class. A BridgeRunner conducts the overall synchronization process. It loads and calls functions on a Connector at the appropriate points in the sequence. The process may be summarized as follows:
+The ultimate purpose of a Connector is to synchronize an iModel with the data in one or more source documents. The synchronization step involves authorization, communicating with an iModel server, converting data, and concurrency control. iTwin.js defines a framework in which the Connector itself can focus on the tasks of extraction, alignment, and change-detection. The other functions are handled by classes provided by iTwin.js. The framework is implemented by the ConnectorRunner class. A ConnectorRunner conducts the overall synchronization process. It loads and calls functions on a Connector at the appropriate points in the sequence. The process may be summarized as follows:
```

Comment:
"iModel server" not sure how i feel about this, but i understand this was what we used before too

Reply:
@aruniverse replaced w iModelHub after discussion w/ my teammates.